shell/platform/tizen/channels/window_channel.cc

```diff
@@ -57,7 +57,7 @@ void WindowChannel::HandleMethodCall(
     EncodableValueHolder<int32_t> width(arguments, "width");
     EncodableValueHolder<int32_t> height(arguments, "height");
 
-    TizenWindow::Geometry geometry = window_->GetWindowGeometry();
+    TizenViewBase::Geometry geometry = window_->GetRenderTargetGeometry();
     // FIXME: Use SetWindowGeometry() instead of OnGeometryChanged()
     // After the SetWindowGeometry was successfully executed, I expected a
```

Comment:
`SetWindowGeometry` has been renamed to `SetRenderTargetGeometry` thus this comment needs an update.